Applications/shapeworks/Commands.cpp

```diff
@@ -44,6 +45,29 @@ bool Example::execute(const optparse::Values &options, SharedCommandData &shared
 }
 #endif
 
+///////////////////////////////////////////////////////////////////////////////
+// Seed
+///////////////////////////////////////////////////////////////////////////////
+void Seed::buildParser()
+{
+  const std::string prog = "seed";
+  const std::string desc = "sets the seed for random number generation";
```

Comment:
please add "(useful for debugging)" to the help string.

Reply:
Done